app/jobs/cache/feed_job.rb

```diff
@@ -19,7 +19,11 @@ def perform(url, download_media = false)
         end
       end
 
-      Rails.cache.write("feed/#{url}", @feed)
+      cached_feed = Rails.cache.fetch("feed/#{url}")
```

Comment:
We have "feed/#{url}" all over the place... could we move that into a new method, `Feed#cache_key`?